packages/web-app-files/src/components/SideBar/SpaceInfo.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="space_info">
+    <div class="space_info__body oc-text-overflow oc-flex oc-flex-middle">
+      <div class="oc-mr-s">
+        <oc-icon name="layout-grid" size="large" class="oc-display-block" />
```

Comment:
Icon size feels too big when you don't have a short description (see screenshot). Worth to choose a smaller size if the space only has a name but no short description?

<img width="479" alt="Screenshot 2022-02-17 at 22 32 39" src="https://user-images.githubusercontent.com/3532843/154574472-9af7ca64-ecb3-4e2a-9d8b-2e2599c43eec.png">